addons/account_asset/wizard/account_asset_change_duration.py

```diff
@@ -43,6 +44,37 @@ def _get_asset_method_time(self, cr, uid, ids, field_name, arg, context=None):
         'asset_method_time': fields.function(_get_asset_method_time, type='char', string='Asset Method Time', readonly=True),
     }
 
+    def fields_view_get(self, cr, uid, view_id=None, view_type='form', context=None, toolbar=False, submenu=False):
+        """ Returns views and fields for current model.
+        @param cr: A database cursor
+        @param user: ID of the user currently logged in
+        @param view_id: list of fields, which required to read signatures
+        @param view_type: defines a view type. it can be one of (form, tree, graph, calender, gantt, search, mdx)
+        @param context: context arguments, like lang, time zone
+        @param toolbar: contains a list of reports, wizards, and links related to current model
+
+        @return: Returns a dictionary that contains definition for fields, views, and toolbars
+        """
+        if not context:
+            context = {}
+        asset_obj = self.pool.get('account.asset.asset')
+        result = super(asset_modify, self).fields_view_get(cr, uid, view_id, view_type, context=context, toolbar=toolbar, submenu=submenu)
+        asset_id = context.get('active_id', False)
```

Comment:
No need of `False` as 2nd parameter, `None` is already the default for `dict.get()`